File: B2083.py
while True:
    line = input()
    if line == "# 0 0":
        break
    name, age, weight = line.split()
    age, weight = int(age), int(weight)

    if age > 17 or weight >= 80:
        print(f"{name} Senior")
    else:
        print(f"{name} Junior")

File: B2083.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

class B2083 {
    public static void main (String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        while (true) {
            String line = br.readLine();
            if ("# 0 0".equals(line)) break;
            String[] splittedLine = line.split(" ");
            String name = splittedLine[0];
            String stringAge = splittedLine[1];
            String stringWeight = splittedLine[2];
            int age = Integer.parseInt(stringAge);
            int weight = Integer.parseInt(stringWeight);

            if (age > 17 || weight >= 80) System.out.println(name + " Senior");
            else System.out.println(name + " Junior");
        }
        br.close();
    }
}